bolt/utils/nfc-check-setup.py

```diff
@@ -156,9 +158,8 @@ def main():
     os.replace(bolt_path, f"{bolt_path}.old")
 
     print(
-        f"Build directory {args.build_dir} is ready to run BOLT tests, e.g.\n"
-        "\tbin/llvm-lit -sv tools/bolt/test\nor\n"
-        "\tbin/llvm-lit -sv tools/bolttests"
+        f"Build directory {args.build_dir} is ready for NFC-Mode comparison "
+        "between the two revisions."
```

Comment:
Keep the example commands? I copy-paste from them often

Reply:
Will do, thanks! Setting up the wrapper now stays under a flag, so I'll reintroduce this example when I rebase the patch.